### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by piuccio**
and instead of depending on the entire `aws-java-sdk`, use only `core` and `s3`.

@jamespamplin

### CODE REVIEW

**Comment by rrees** [comment]
I'm not sure this is a sensible change as now you have two dependencies that need to change in lockstep. If you are going to make this change abstract out the AWS SDK and Akka version number into values that are easy to change without having to go through and change the individual dependencies.

File: build.sbt [hunk 1]
```diff
@@ -2,22 +2,23 @@ name := """editorial-permissions-client"""
...
 
 libraryDependencies ++= Seq(
-  "com.amazonaws" % "aws-java-sdk" % "1.9.29",
-  "com.typesafe.akka" %% "akka-actor" % "2.3.4",
-  "com.typesafe.akka" %% "akka-agent" % "2.3.4",
-  "com.typesafe.akka" %% "akka-slf4j" % "2.3.4",
-  "net.liftweb" %% "lift-json" % "2.6",
-  "org.mockito" % "mockito-all" % "1.8.5" % "test",
-  "org.scalatest" %% "scalatest" % "2.2.4" % "test"
+  "com.amazonaws" % "aws-java-sdk-core" % "1.10.37",
```
Comment (jamespamplin) [inline build.sbt]: Thanks for this! It looks like [`aws-java-sdk-s3`](http://search.maven.org/#artifactdetails|com.amazonaws|aws-java-sdk-s3|1.10.37|jar) already depends on `aws-java-sdk-core`, so we could remove this explicit dependency here too.

**Comment by piuccio** [comment]
@rrees last commit should do

**Comment by rrees** [comment]
:+1:

**Comment by jamespamplin** [comment]
Thanks @piuccio - published as v0.3

Should be available in an hour or so.

**Comment by sihil** [comment]
@jamespamplin No 2.10?

**Comment by sihil** [comment]
Oh, I see, it was switched to 2.11.

**Comment by jamespamplin** [comment]
@sihil yeah this upgrade of dependencies removes support for 2.10. Probably should remove the cross-compile config and update the readme too.

**Pull request merged**